public_data/models.py

```diff
@@ -34,10 +34,6 @@ def public_count(project):
             project_membership__project=project,
             # Filter to only count members with datafiles for this project.
             is_public=True,
-            project_membership__project__in=F(
```

Comment:
For some reason this kept failing, as somewhere along the way the way these interpreted are changed. Before the SQL query ended up being this

```
SELECT "public_data_publicdataaccess"."id", "public_data_publicdataaccess"."participant_id", "public_data_publicdataaccess"."project_membership_id", "public_data_publicdataaccess"."is_public" FROM "public_data_publicdataaccess" INNER JOIN "private_sharing_datarequestprojectmember" ON ("public_data_publicdataaccess"."project_membership_id" = "private_sharing_datarequestprojectmember"."id") INNER JOIN "open_humans_member" ON ("private_sharing_datarequestprojectmember"."member_id" = "open_humans_member"."id") INNER JOIN "auth_user" ON ("open_humans_member"."user_id" = "auth_user"."id") INNER JOIN "data_import_datafile" ON ("auth_user"."id" = "data_import_datafile"."user_id") INNER JOIN "private_sharing_projectdatafile" ON ("data_import_datafile"."id" = "private_sharing_projectdatafile"."parent_id") WHERE ("public_data_publicdataaccess"."is_public" = True AND "private_sharing_datarequestprojectmember"."project_id" = 92 AND "private_sharing_datarequestprojectmember"."project_id" IN ("private_sharing_projectdatafile"."direct_sharing_project_id"))
```
(note the parentheses for the last `AND`), with the updates here it ended up being 

```
SELECT "public_data_publicdataaccess"."id", "public_data_publicdataaccess"."participant_id", "public_data_publicdataaccess"."project_membership_id", "public_data_publicdataaccess"."is_public" FROM "public_data_publicdataaccess" INNER JOIN "private_sharing_datarequestprojectmember" ON ("public_data_publicdataaccess"."project_membership_id" = "private_sharing_datarequestprojectmember"."id") INNER JOIN "open_humans_member" ON ("private_sharing_datarequestprojectmember"."member_id" = "open_humans_member"."id") INNER JOIN "auth_user" ON ("open_humans_member"."user_id" = "auth_user"."id") INNER JOIN "data_import_datafile" ON ("auth_user"."id" = "data_import_datafile"."user_id") INNER JOIN "private_sharing_projectdatafile" ON ("data_import_datafile"."id" = "private_sharing_projectdatafile"."parent_id") WHERE ("public_data_publicdataaccess"."is_public" AND "private_sharing_datarequestprojectmember"."project_id" = 92 AND "private_sharing_datarequestprojectmember"."project_id" IN "private_sharing_projectdatafile"."direct_sharing_project_id")
```

Given that the only effect is "miscounting" the number of users (by including those that do have no files but are willing to make them public in principle) I think that's acceptable